src/app/components/onboarding/profile/profile.component.html

```diff
@@ -12,7 +12,7 @@ <h2>{{ 'PROFILE_TITLE' | translate }}</h2>
       <form [formGroup] = "profileFormGroup" (ngSubmit)="onSubmit()">
          <div *ngFor="let control of profileFormGroup.controls | keyvalue : returnZero">
           <p class="p-float-label">
-            <input id="{{control.key}}" type="text" pInputText formControlName="{{control.key}}"> 
+            <input id="{{control.key}}" type="text" pInputText formControlName="{{control.key}}">
```

Comment:
Hi, warum sind in deinem PR Anpassungen drin für die Profile Component?

Reply:
diese anpassung stammt nicht von mir (also zeile 15)

Reply:
@wolfgangroese Ich glaube Mira meint die gesamten Änderungen. Auch die Nachfolgenden bezüglich der QR Code Komponente. Meine Dummy Komponenten von letztens sind natürlich kein Glanzbeispiel für ordentliche Software Entwicklung :D.